guides.md

```diff
@@ -190,6 +191,43 @@ We'll now send your first email to Mailinator, a cloud inbox that's helpful for
 9. Visit [the corresponding cloud inbox](https://www.mailinator.com/v3/index.jsp?zone=public&query=hyphacoop) and confirm your message has arrived, It might take a few minutes.
 10. Don’t forget to email Infra WG as mentioned in onboarding email.
 
+## Expense Reimbursement
+
+Employees should try to submit eligible expenses in the same quarter they are incurred.
+The Finance WG reviews submitted expenses before each pay period, and pays out reimbursement amounts on the next pay day.
+
+### Submitting an expense
+
+1. Upload the expense receipt to [Employee Expenses](https://link.hypha.coop/receipts) with file name `xx-YYYY-MM-DD.ext` where `xx` is the initials of the member and `ext` the file extension, then make note of the URL.
+
+1. Open the [Expense Reimbursement](https://link.hypha.coop/expenses) sheet and fill in a new row according to instructions in the sheet.
+    - Date of Expense: `[date on your receipt]`
+    - Employee: `[employee to reimburse]`
+    - Description of Expense: `[short description of what the expense is for]`
+    - Total Amount: `[total receipt amount, including HST]`
+    - HST Amount: `[total HST on the receipt]` 
+    - Currency: `[currency of the receipt]` (default to `CAD`)
+    - Exchange Rate: `[currency exchange rate to CAD]` (default to `1.000` for CAD)
```

Comment:
How will people substantiate an exchange rate? Do they have to show that portion of CC statement?

Reply:
I think it can't be that far off to publicly kept rates, and I'd rather not add overhead?

Reply:
The rates change from the time its posted on my CC to when my statement comes out. market rates are pretty far from bank rates

Reply:
Members should put the rate they actually paid out of CC, but we can leave this to the discretion of the member. If it's not unreasonable, Finance will approve it and not ask employee for proof.

There are many scenarios where it's not a CC and the rate isn't well established. For example, when I travel I mostly pay cash, and have had to use public rates / averaged rates for claims.

```suggestion
    - Exchange Rate: `[currency exchange rate to CAD]` (default to `1.000` for CAD, you use the rate on your credit card statement or market rate on the receipt date)
```

Reply:
Discussed with @YurkoWasHere we won't enforce uploading of CC statement.